manifests/07-operator.yaml

```diff
@@ -55,3 +56,28 @@ spec:
               value: "cluster-image-registry-operator"
             - name: IMAGE
               value: docker.io/openshift/origin-docker-registry:latest
+        - name: cluster-image-registry-operator-watch
+          image: docker.io/openshift/origin-cluster-image-registry-operator:latest
+          command:
+          - cluster-image-registry-operator-watch
+          - file-watcher-watchdog
+          args:
+          - --namespace=$(POD_NAMESPACE)
+          - --process-name=cluster-image-registry-operator
+          - --termination-grace-period=30s
+          - --files=/usr/share/pki/ca-trust-source/ca-bundle.pem
```

Comment:
/hold

Mounted file name may change.

Reply:
@ricardomaraschini change the file to `/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem`, and we can merge this.

Reply:
@adambkaplan done.